src/views/ManageList.jsx

```diff
@@ -1,7 +1,80 @@
-export function ManageList() {
+import { addItem } from '../api/firebase';
+import { useRef } from 'react';
+
+export function ManageList({ listPath }) {
+	const formRef = useRef();
+
+	const handleSubmit = async (e) => {
+		e.preventDefault();
+
+		const formData = new FormData(e.target);
+		const itemName = formData.get('itemName');
+		const daysUntilNextPurchase = parseInt(
+			formData.get('daysUntilNextPurchase'),
+		);
+
+		try {
+			await addItem(listPath, {
+				itemName,
+				daysUntilNextPurchase,
+			});
+
+			alert('Item saved successfully');
+		} catch (error) {
+			alert('There was a problem');
+		}
+
+		formRef.current.reset();
+	};
+
 	return (
-		<p>
-			Hello from the <code>/manage-list</code> page!
-		</p>
+		<form ref={formRef} onSubmit={handleSubmit}>
+			<div>
+				<label htmlFor="item">Item Name</label>
+				<input name="itemName" type="text" id="item" required />
+			</div>
+
+			<div>
+				<label htmlFor="daysUntilNextPurchase">
+					How soon do you think you'll purchase this item again?
+				</label>
+
+				<p>Please select an option:</p>
+
+				<ul>
+					<li>
+						<label htmlFor="soon"> Soon (7 days)</label>
+						<input
+							value={7}
+							name="daysUntilNextPurchase"
+							type="radio"
+							id="soon"
+							required
```

Comment:
I said this to the other group as well, but the built in HTML validations actually aren't as accessible as one would hope. So I think down the road we should build our own validation + status displaying logic.